suite-native/intl/src/en.ts

```diff
@@ -11,6 +11,16 @@ export const en = {
             confirm: 'Confirm',
             continue: 'Continue',
             next: 'Next',
+            dismiss: 'Dismiss',
+        },
+        unknownError: 'Something went wrong',
```

Comment:
unknownError is duplicated. It is now both in `generic` and `messages`
```suggestion
```

Reply:
9860215781